endo_api/settings_prod.py

```
@@ -0,0 +1,84 @@
+db_name = db_cfg.name
+
+
+ASSET_DIR = Path(__file__).parent / "tests/assets"
+RUN_VIDEO_TESTS = os.environ.get("RUN_VIDEO_TESTS", "true").lower() == "true"
+
+# Production settings
+DEBUG = os.environ.get("DJANGO_DEBUG", "False")  # Changed from True to False for production
+if DEBUG.lower() == "true":
+    DEBUG = True
+else:
+    DEBUG = False
```

Ruff (0.12.2)

Use ternary operator `DEBUG = True if DEBUG.lower() == "true" else False` instead of `if`-`else`-block

Replace `if`-`else`-block with `DEBUG = True if DEBUG.lower() == "true" else False`

(SIM108)